resources/lang/fr/profile_lost_person.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+return [
+    'development_disorder'            => 'Trouble du développement',
+    'alzheimer_or_other_dementias'    => 'Alzheimer ou autres démences',
+    'mental_or_psychological_illness' => 'Troubles psychologiques',
+    'autolithic_behaviors'            => 'Comportements autolithiques',
+    'hiker'                           => 'Randonneur',
+    'collector'                       => 'Collectionneur'
```

Comment:
A comma is missing after 'Collectionneur'

Reply:
Fixed in https://github.com/eduayme/RescueApp/pull/148/commits/794fd5ca5d8261cc804a9d362527f4f918781090